src/algebra/continued_fractions/computation/approximations.lean

```diff
@@ -0,0 +1,279 @@
+/-
+Copyright (c) 2020 Kevin Kappelmann. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Kappelmann
+-/
+import algebra.continued_fractions.computation.translations
+import algebra.continued_fractions.continuants_recurrence
+import algebra.continued_fractions.terminated_stable
+import data.nat.fib
+import tactic.lint
```

Comment:
```suggestion

```

Reply:
There should be a lint that reminds me to remove the lint import :stuck_out_tongue_closed_eyes: